WcaOnRails/app/mailers/competitions_mailer.rb

```diff
@@ -8,7 +8,7 @@ def notify_board_of_confirmed_competition(confirmer, competition)
     @confirmer = confirmer
     mail(
       to: "board@worldcubeassociation.org",
-      cc: competition.delegates.pluck(:email),
+      cc: competition.delegates.pluck(:email) + competition.delegates.map { |d| d.senior_delegate&.email }.compact.flatten(1),
```

Comment:
Whoops, you're right, that flatten is pointless! I'll change it to a `compact.uniq` =)